src/Plugin/ConceptSubset/DepartmentsAgencies.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_corporate_site_info\Plugin\ConceptSubset;
+
+use Drupal\Core\Entity\Query\QueryInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\rdf_entity\RdfFieldHandlerInterface;
+use Drupal\rdf_skos\ConceptSubsetPluginBase;
+use Drupal\rdf_skos\Plugin\PredicateMapperInterface;
+
+/**
+ * Creates a subset of the corporate bodies vocabulary.
+ * @codingStandardsIgnoreStart
```

Comment:
How come this was never a CS problem when declaring other plugins or even this plugin type elsewhere?

Reply:
I had this phpsc message and it is connected to very long plugin id (with added module name prefix).
` 18 | WARNING | Line exceeds 80 characters; contains 82 characters`